### DISCUSSION TITLE
Erasure Code Calculator and stripe size

### DISCUSSION THREAD

**Discussion by sensille** [minio/minio#17336] [Q&A] [answered]
With a given configuration, the calculator gives me several options for the stripe size (K+M), for example 12, 9, 6 and 3. I know how to configure/choose the parity (M), but how can I tell the server which K+M to use?
Other parts of the documentation refer to the process as "automatic", but I want to use a different K+M than the default one.

**Comment by klauspost** [comment, marked as answer]
@sensille We do not recommend changing the default set/stripe size, as it has significant impact on your availability.

We can through a setup review as part of our [subscriptions](https://min.io/pricing). Feel free to reach out on `hello@min.io` if you are doing a POC.

The `MINIO_ERASURE_SET_DRIVE_COUNT` environment variable controls it - and it must be configured before first startup. Selecting the "correct" value is complex and you can easily end up with a sub-par configured cluster.
